app/mailers/vacation_mailer.rb

```diff
@@ -0,0 +1,6 @@
+class VacationMailer < ApplicationMailer
+  def notify_vacation_request(vacation, admins_emails)
+    @vacation = vacation
+    mail(to: admins_emails, subject: (t '.subject', user: @vacation.user.name))
```

Comment:
```suggestion
    mail(to: admins_emails, subject: t('.subject', user: @vacation.user.name))
```
Just a suggestion :nail_care: